### PR TITLE
Update httproute.md to add link for v1.SectionName

### PR DESCRIPTION
**Pull request by fatsheep9146**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

> Rules- Define a list of rules to perform actions against matching HTTP requests. Each rule consists of matches, filters (optional), backendRefs (optional), timeouts (optional), and [name][name] (optional) fields.

In this line, the link called `name` is not added.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: site-src/api-types/httproute.md [hunk 1]
```diff
@@ -339,3 +339,4 @@ resolution applies to merging, refer to the [API specification][httprouterule].
 [parentRef]: /reference/spec/#gateway.networking.k8s.io/v1.ParentRef
 [timeouts]: /reference/spec/#gateway.networking.k8s.io/v1.HTTPRouteTimeouts
 [appProtocol]: https://kubernetes.io/docs/concepts/services-networking/service/#application-protocol
+[name]: /reference/spec/#gateway.networking.k8s.io/v1.SectionName
```
Comment (robscott) [inline site-src/api-types/httproute.md:342]: Nit: Prefer "sectionName" to be more specific (would also require change where this is referenced)
```suggestion
[sectionName]: /reference/spec/#gateway.networking.k8s.io/v1.SectionName
```

**Review by robscott — COMMENTED** [review]
Thanks @fatsheep9146!

File: site-src/api-types/httproute.md (hunk 1 above)
Comment (fatsheep9146) [reply site-src/api-types/httproute.md]: > Nit: Prefer "sectionName" to be more specific (would also require change where this is referenced)

Done

**Comment by robscott** [comment]
Thanks!

/lgtm

**Pull request merged**